bpi.syndicate.inc

```diff
@@ -120,16 +230,16 @@ function bpi_syndicate_page_form_submit($form, &$form_state) {
  *   Page offset.
  * @param array $filters
  *   Array of filters/sorting.
- * @param string $sort
+ * @param array $sort
  *   Sorting order.
  *
  * @return \Bpi\Sdk\NodeList|array
  *   Search result items.
  */
-function bpi_search_content($query = '', $page = 1, $filters = array(), $sort = 'desc') {
+function bpi_search_content($query = '', $page = 1, $filters = array(), $sort = array()) {
   try {
     $bpi = bpi_client_instance();
-    $amount = variable_get('bpi_content_per_page', BPI_CONTENT_PER_PAGE);
+    $amount = max(intval(_bpi_get_query_parameter('amount', 10)), 10);
```

Comment:
Amount or page size is hardcoded to 10, maybe it should be in an administrator UI form?